src/main/java/uk/gov/hmcts/reform/sendletter/controllers/SendLetterController.java

```diff
@@ -125,6 +125,19 @@ public SendLetterController(
         return noContent().build();
     }
 
+    @PostMapping(path = "/check-print-state")
+    @ApiOperation(value = "Check print status after parsing the report from Xerox")
+    public ResponseEntity<Void> checkPrintState(
```

Comment:
can we move it to a different controller may be StatusCheckerController? what are your thoughts?

Reply:
good idea.
since all endpoints are pretty minimum, maybe just wait for similar one to come in (if any) and then refactor if necessary.

Reply:
what is going to call this endpoint? what's the general solution?

Reply:
there is a jira link explaining the story. basically after report about print state is finished, need to collect information of what's not yet printed and report to app insights

Reply:
going with a separate controller (as @nitinprabhu suggested) would make this api in line with hmcts guidelines:

>Must: Keep URLs Verb-Free
https://hmcts.github.io/restful-api-standards/#141